css/home/features.css

```diff
@@ -25,3 +25,27 @@
 .features-main .grid > *:nth-child(2) {
   grid-column: 1 / span 2;
 }
+
+@media all and (max-width: 768px) {
+    
+  .features-main .grid  {
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+  }
+  .features-head,
+  .features-sub-head  {
+    text-align: center;
+  }
+
+  .features-head img,
```

Comment:
Ohhhh

Reply:
Esto es de features.. solo he trabajo en Index, recuerdo haberte comentado que mejor hago todo... si cambio solo el img de feature, tendre que refactorizar con  BEM todo, features y docs, y no lo voy a hacer.

Reply:
Ok but features.css should not be in the directory of home because it is not related to the homepage that's why it should be on another directory (for example: css/features). 

**This needs to be changed for the future, every file should be related to the page not another page.**